src/components/Catalog.jsx

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from 'react'
+import Movie from './subComponets/Movie'
+import SearchItem from './subComponets/searchItem'
+import './style/catalog.css'
+import { Link } from 'react-router-dom'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faHome, faLongArrowAltLeft } from '@fortawesome/free-solid-svg-icons'
+class Catalog extends Component {
+    constructor() {
+        super()
+        this.state = {
+            term: "",
+            filteredMovies: []
+        }
+    }
+
+    handleInput = (e) => {
```

Comment:
you can deconstruct it
`handleInput = ({target}) => { const term = target.value.toLowerCase()...`